packages/cli/tests/unit/fixtures/javascript/yarn_with_subprojects/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "yarn_with_subprojects",
+  "packageManager": "yarn@3.2.1",
+  "private": true,
+  "workspaces": [ "packages/*" ]
+}
```

Comment:
I wonder if you even need this file for if you're mocking the yarn outputs anyway?

Reply:
I used it to run the installer manually.
```
npm start install -- -d /home/test/src/appmap-js/packages/cli/tests/unit/fixtures/javascript/yarn_with_subprojects
```
It didn't help much because I got the [error](https://user-images.githubusercontent.com/111290954/211630706-b129b9f5-8019-40c9-8603-eb36defd303a.png) in the OP.   Maybe it'd be helpful to other devs.